samples/apps/copilot-chat-app/webapi/PlannerSkills/Notegen/skprompt.txt

```diff
@@ -0,0 +1,21 @@
+Analyze the following extract taken from a document. 
+- Produce key points for memory. 
+- Give memory a name. 
+- Extract only points worth remembering. 
+- Be brief. Conciseness is very important.  
+- Use broken English. 
```

Comment:
This could be interpreted in different ways.  Maybe the following is more like what you want?

```suggestion
- Use shorthand notation.
```

Reply:
grammatically incomplete?

Reply:
removed